Libraries/Inspector/ElementBox.js

```diff
@@ -31,6 +32,8 @@ class ElementBox extends React.Component<$FlowFixMeProps> {
     };
 
     if (margin != null) {
+      resolveRelativeSizes(margin);
```

Comment:
Could you change this API to return a new object, like `margin = resolveRelativeSizes(margin)`? I think that will be clearer.